app/assets/stylesheets/modules/review.scss

```diff
@@ -0,0 +1,51 @@
+@import "variables";
+@import "mixins";
+
+#reviews-column {
+  @include padding;
+
+  .reviews {
+    background-color: #FFF;
+    border: none;
+    width: 80%;
+
+    .small-6 {
+      padding: 0;
+      margin:0;
+
+      img {
+        @include image-style;
+        height: 5em;
+        width: 5em;
+      }
+
+      .row-name {
+          font-size: 0.9em;
+          text-align: center;
+      }
+    }
+
+    #review-rating {
```

Comment:
Avoid using id selectors<br>Selector should have depth of applicability no greater than 2, but was 3